apps/src/lib/ui/WireframeButtons.jsx

```diff
@@ -111,6 +111,7 @@ SendToPhoneButton.propTypes = {
   onClick: PropTypes.func.isRequired
 };
 
+/* eslint-disable react/prop-types */
```

Comment:
NIT: Might be nice to add an additional comment explaining why we are disabling this lint error.

Reply:
Done!